egs/iam/v2/local/chain/tuning/run_cnn_e2eali_1d.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# e2eali_1c is the same as e2eali_1c but has more CNN layers, different filter size
```

Comment:
e2eali_1c is --> e2eali_1d is
less minibatch, frams-per-iter --> smaller minibatches and frames-per-iter